spec/support/boxes/model_with_default_values_box.cr

```diff
@@ -0,0 +1,4 @@
+class ModelWithDefaultValuesBox < BaseBox
```

Comment:
I'm having trouble understanding this part. Maybe you could use the operation directly without any args instead of having a box? Since boxes usually set fake data it seems a bit odd to have one that doesn't do anything